Assets/Group1/Scripts/Enemy.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+
+public class Enemy : MonoBehaviour
+{
+    [SerializeField] private float _deathRange;
+
+    public event UnityAction EnemyDyed;
```

Comment:
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods